### PR TITLE
2218 Apply new manifest type to Breadboard Web `local-graphs.json`

### PR DESCRIPTION
**Pull request by Mearman**
resolves https://github.com/breadboard-ai/breadboard/issues/2218

### CODE REVIEW

File: packages/breadboard-web/index.html [hunk 1]
```diff
@@ -9,7 +9,6 @@
 </head>
 <body>
   <script type="module">
-    import * as pkg from "./package.json";
```
Comment (dglazkov) [inline packages/breadboard-web/index.html:12]: Isn't this used to get the version to print in the console?

File: packages/manifest/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (dglazkov) [inline packages/manifest/.vscode/settings.json:1]: We don't need this, right?

File: packages/breadboard-web/src/providers/examples.ts [hunk 3]
```diff
@@ -153,3 +164,15 @@ export class ExamplesGraphProvider implements GraphProvider {
     );
   }
 }
+/**
+ *
+ * @deprecated this works for title generation but in BB web when you click on the example it will throw an error
```
Comment (dglazkov) [inline packages/breadboard-web/src/providers/examples.ts:169]: Maybe remove?

File: packages/breadboard-web/index.html (hunk 1 above)
Comment (Mearman) [reply packages/breadboard-web/index.html:12]: oops that got in from a branch switch

File: packages/breadboard-web/index.html [hunk 4]
```diff
@@ -9,7 +9,6 @@
 </head>
 <body>
   <script type="module">
```
Comment (Mearman) [inline packages/breadboard-web/index.html:11]:
```suggestion
  <script type="module">
    import * as pkg from "./package.json";
```

File: packages/breadboard-web/src/providers/examples.ts [hunk 5]
```diff
@@ -10,36 +10,47 @@ import {
...
-        .sort((a, b) => a.title.localeCompare(b.title))
-        .map((board) => {
-          return [
-            board.title,
-            { url: board.url, readonly: true, mine: false, handle: undefined },
-          ];
-        })
+        .map((board) => ({
+          ...board,
+          title: board.title || board.reference || hashOf(board),
```
Comment (Mearman) [inline packages/breadboard-web/src/providers/examples.ts:42]:
```suggestion
          title: board.title || board.reference || "",
```

File: packages/breadboard-web/src/providers/examples.ts [hunk 6]
```diff
@@ -153,3 +164,15 @@ export class ExamplesGraphProvider implements GraphProvider {
...
+ * @deprecated this works for title generation but in BB web when you click on the example it will throw an error
+ */
+function hashOf(obj: object): string {
+  const stringified = JSON.stringify(obj);
+  const hash = stringified.split("").reduce((acc, char) => {
+    const charCode = char.charCodeAt(0);
+    return (acc << 5) - acc + charCode;
+  }, 0);
+  return hash.toString(16);
+}
```
Comment (Mearman) [inline packages/breadboard-web/src/providers/examples.ts:178]:
```suggestion
```

**Review by dglazkov — APPROVED** [review]

**Pull request merged**